ibm/service/kubernetes/resource_ibm_container_dedicated_host_pool.go

```diff
@@ -122,15 +134,22 @@ func resourceIBMContainerDedicatedHostPoolCreate(d *schema.ResourceData, meta in
 
 	res, err := dedicatedHostPoolAPI.CreateDedicatedHostPool(params, targetEnv)
 	if err != nil {
-		return fmt.Errorf("[ERROR] Error creating host pool %v", err)
+		return diag.Errorf("[ERROR] Error creating host pool %v", err)
 	}
 
 	d.SetId(res.ID)
 
-	return resourceIBMContainerDedicatedHostPoolRead(d, meta)
+	dhp, err := waitForDedicatedHostPoolAvailable(ctx, dedicatedHostPoolAPI, res.ID, DedicatedHostCreateTimeout, targetEnv)
```

Comment:
use schema.TimeoutCreate (respective timeout) instead of DedicatedHostCreateTimeout  in all wait logics

Reply:
done